pkg/cmd/cli/cmd/expose.go

```diff
@@ -56,7 +56,13 @@ func NewCmdExpose(fullName string, f *clientcmd.Factory, out io.Writer) *cobra.C
 		cmdutil.CheckErr(err)
 		defRun(cmd, args)
 	}
-	cmd.Flags().String("hostname", "", "Set a hostname for the new route")
+	cmd.Flags().String("hostname", "", "Set a hostname for the new route. Optional.")
+	cmd.Flags().String("path", "", "Path that the router watches to route traffic to the service. Optional.")
+	cmd.Flags().String("tls-termination", "", "Indicate a termination type for the generated route. If not set, any termination config will be ignored. Supports 'edge', 'passthrough', or 'reencrypt'.")
```

Comment:
Add an option for `insecureEdgeTerminationPolicy` - supported values are `Allow` | `Redirect` | `None'. 
Currently this is only valid for edge terminated routes. Will need to add passthrough routes at some point soon.
Though when we do that, the name with the Edge in it (and no `Bono`!!) might be a bit wonky ... maybe we will learn how to dismantle that (obligatory ref: http://www.amazon.com/How-Dismantle-Atomic-Bomb-U2/dp/B0006399FS) ... ;^)